common/main/sr_Latn.xml

```diff
@@ -2875,22 +2878,22 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 				<relative type="2">prekosutra</relative>
 				<relativeTime type="future">
 					<relativeTimePattern count="one">za {0} dan</relativeTimePattern>
-					<relativeTimePattern count="few">za {0} dana</relativeTimePattern>
+					<relativeTimePattern count="few">↑↑↑</relativeTimePattern>
 					<relativeTimePattern count="other">za {0} dana</relativeTimePattern>
 				</relativeTime>
 				<relativeTime type="past">
-					<relativeTimePattern count="one">pre {0} dana</relativeTimePattern>
-					<relativeTimePattern count="few">pre {0} dana</relativeTimePattern>
+					<relativeTimePattern count="one">↑↑↑</relativeTimePattern>
+					<relativeTimePattern count="few">↑↑↑</relativeTimePattern>
 					<relativeTimePattern count="other">pre {0} dana</relativeTimePattern>
 				</relativeTime>
 			</field>
 			<field type="day-short">
 				<displayName>d.</displayName>
-				<relative type="-2">prekjuče</relative>
```

Comment:
This is very suspicious. sr_Latn inherits directly from root, so replacing by ↑↑↑ means throwing away prekjuče and replacing by a root value.

Reply:
🤔

Reply:
@macchiati two things:

1. this conforms to what's in `sr.xml` (see below)
2. this is `day-short` - so it will inherit from `day` which still has these values.


`sr.xml`: 

```xml
			<field type="day">
				<displayName>дан</displayName>
				<relative type="-2">прекјуче</relative>
				<relative type="-1">јуче</relative>
				<relative type="0">данас</relative>
				<relative type="1">сутра</relative>
				<relative type="2">прекосутра</relative>
				<relativeTime type="future">
					<relativeTimePattern count="one">за {0} дан</relativeTimePattern>
					<relativeTimePattern count="few">↑↑↑</relativeTimePattern>
					<relativeTimePattern count="other">за {0} дана</relativeTimePattern>
				</relativeTime>
				<relativeTime type="past">
					<relativeTimePattern count="one">↑↑↑</relativeTimePattern>
					<relativeTimePattern count="few">↑↑↑</relativeTimePattern>
					<relativeTimePattern count="other">пре {0} дана</relativeTimePattern>
				</relativeTime>
			</field>
			<field type="day-short">
				<displayName>д.</displayName>
				<relative type="-2">↑↑↑</relative>
				<relative type="-1">↑↑↑</relative>
				<relative type="0">↑↑↑</relative>
				<relative type="1">↑↑↑</relative>
				<relative type="2">↑↑↑</relative>
```

`sr_Latn.xml`: 

```xml
			<field type="day">
				<displayName>dan</displayName>
				<relative type="-2">prekjuče</relative>
				<relative type="-1">juče</relative>
				<relative type="0">danas</relative>
				<relative type="1">sutra</relative>
				<relative type="2">prekosutra</relative>
```

Reply:
Ok, thought it over and that will work. Thanks for the explanation.